spring-cloud-kubernetes-client-config/src/test/java/org/springframework/cloud/kubernetes/client/config/Constants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.kubernetes.client.config;
+
+/**
+ * A common place for some constants used in tests.
+ *
+ * @author wind57
+ */
+public final class Constants {
```

Comment:
more things will come in here in time, and most probably will be moved to commons, but sometime later

Reply:
I would rather put it in commons now, we wont be able to move it later

Reply:
now that I think about it - we do not even need this one. it is in `/src/test/java`, so not that of a real value. I deleted it.